source/object-retention/minio-object-locking.rst

```diff
@@ -312,41 +300,39 @@ preferred SDK.
 Enable Legal Hold Retention
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can enable or disable indefinite Legal Hold retention for an object using
-the MinIO Console, the MinIO :mc:`mc` CLI, or using an S3-compatible SDK. 
-You can place a legal hold on an object already held under a 
-:ref:`COMPLIANCE <minio-object-locking-compliance>` or 
-:ref:`GOVERNANCE <minio-object-locking-governance>` lock. The object remains
-WORM locked until the retention lock expires *and* the legal hold is lifted.
+You can enable or disable indefinite Legal Hold retention for an object using the MinIO Console, the MinIO :mc:`mc` CLI, or using an S3-compatible SDK. 
+
+You can place a legal hold on an object already held under a :ref:`COMPLIANCE <minio-object-locking-compliance>` or :ref:`GOVERNANCE <minio-object-locking-governance>` lock. 
+The object remains WORM locked under the legal hold even when the retention lock expires. 
+You or another user with the necessary permissions must explicitly lift the legal hold to remove the WORM lock.
 
 .. tab-set::
 
    .. tab-item:: MinIO Console
       :sync: console
 
-      Select the :guilabel:`Object Browser` section of the MinIO Console. Select
-      the bucket row from the list of buckets. You can use the :octicon:`search`
-      :guilabel:`Search` bar to filter the list. 
-      
-      .. image:: /images/minio-console/console-object-browser-locking.png
+      Select the :guilabel:`Buckets` section of the MinIO Console to access bucket creation and management functions. 
+      You can use the :octicon:`search` :guilabel:`Search` bar to filter the list. 
+
+      .. image:: /images/minio-console/console-bucket.png
          :width: 600px
          :alt: MinIO Console Bucket Management
          :align: center
 
-      Browse to the object and select it to open the object details view. 
-      Click the :octicon:`pencil` icon on the :guilabel:`Legal Hold` row to
-      toggle the Legal Hold status of the object.
+      Each bucket row has a :guilabel:`Manage` button that when selected opens the management view for that bucket.
 
-      .. image:: /images/minio-console/console-object-browser-object-details.png
+      .. image:: /images/minio-console/console-object-browser.png
          :width: 600px
-         :alt: MinIO Console Bucket Default Retention
+         :alt: MinIO Console Bucket Object Browser
          :align: center
 
+      Browse to the object and select it to open the object details view. 
+      Select the :guilabel:`Legal Hold` button to toggle the Legal Hold status of the object.
+
    .. tab-item:: MinIO CLI
       :sync: cli
 
-      Use The command command to enable or disable the legal
-      hold on an object.
+      Use the :guilabel:`mc legalhold set` command to toggle the legal hold status on an object.
 
```

Comment:
Line 427:
Either drop the `the` before the permission name, or add the word `permission` to make it a modifier.

```
Only privileged users with the :policy-action:`s3:PutObjectLegalHold` permission can set or
```

Reply:
adding permission makes more sense.